mailer/models.py

```diff
@@ -43,21 +44,13 @@ def retry_deferred(self, new_priority=2):
 def email_to_db(email):
     # pickle.dumps returns essentially binary data which we need to encode
     # to store in a unicode field.
-    return base64.encodestring(pickle.dumps(email))
+    return force_text(base64.encodestring(pickle.dumps(email)))
 
 
 def db_to_email(data):
-    if data == u"":
+    if not data:
         return None
-    else:
-        try:
-            return pickle.loads(base64.decodestring(data))
-        except Exception:
-            try:
-                # previous method was to just do pickle.dumps(val)
-                return pickle.loads(data.encode("ascii"))
-            except Exception:
-                return None
+    return pickle.loads(base64.decodestring(smart_bytes(data)))
```

Comment:
`base64.decodestring` expects a `byte` string, so we are doing that.

Reply:
Do you know what this exception was for? And will it ever happen with these changes?

Reply:
It was for backwards compatibility. And since we go through all of our messages all the time, we don't need the fallback.